ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,405 @@
+public protocol CardDelegate {
+    /// Called after the Card or the button inside the Card is tapped
+    /// - Parameter card: The Card that was tapped
+    @objc optional func didTapCard(_ card: CardView)
+}
+
+/// Color theme can be app colors, neutral colors, or custom colors. The theme affects the background, border, title, subtitle and icon colors
+@objc(MSFCardColorTheme)
+public enum CardColorTheme: Int, CaseIterable {
+    case appColor
+    case neutral
+    case custom
+}
+
+/// Card style can be horizontal, or vertical
+/// Both styles have a title, an icon and an optional subtitle. The whole card is clickable
+@objc(MSFCardStyle)
+public enum CardStyle: Int, CaseIterable {
+    /// Icon is positioned next to the title and subtitle is positioned below the title
+    case horizontal
+    /// Icon is at the top, title below the icon and subtitle below the title
+    case vertical
+
+    var width: CGFloat {
+        switch self {
+        case .horizontal:
+            return 156
+        case .vertical:
+            return 120
+        }
+    }
+}
+
+/**
+ `CardView` is a UIView used to display information in a card
+ 
+ A card has a title, an optional subtitle, an icon, a style, and a color theme
+ 
+ Use `titleNumberOfLines` and `subtitleNumberOfLines`  to set the number of lines the title and subtitle should have respectively. When the string can not fit in the number of lines set, it will get truncated
+ 
+ Use one of the defined color themes for an app color theme, or a neutral gray color theme
+ When CardColorTheme.custom is used, default colors will be set unless a custom color is provided by setting the properties: `customBackgroundColor`, `customTitleColor`, `customSubtitleColor`, `customIconTintColor`, and `customBorderColor`
+ 
+ Conform to the `CardDelegate` in order to provide a handler for the card tap event
+ */
+@objc(MSFCardView)
+open class CardView: UIView {
+    /// Delegate to handle user interaction with the CardView
+    @objc public weak var delegate: CardDelegate?
+    /// All card styles have a title. Setting `primaryText` will refresh the layout constraints
+    @objc open var primaryText: String {
+        didSet {
+            if primaryText != oldValue {
+                primaryLabel.text = primaryText
+                setupLayoutConstraints()
+            }
+        }
+    }
+    /// Setting `secondaryText` is a way to add/remove the subtitle which will also refresh the layout constraints to adjust to the change
+    @objc open var secondaryText: String? {
+        didSet {
+            if secondaryText != oldValue {
+                secondaryLabel.text = secondaryText
+                setupLayoutConstraints()
+            }
+        }
+    }
+    /// The card's icon. Announcement is the only card style that doesn't include an icon
+    @objc open var icon: UIImage {
+        didSet {
+            if icon != oldValue {
+                iconView.image = icon
+            }
+        }
+    }
```

Comment:
nit: you should add empty lines throughout this change between properties and methods.